runtime-sdk/modules/evm/src/test.rs

```diff
@@ -1201,3 +1207,527 @@ fn test_return_value_limits() {
     assert_eq!(result[64], 0xFF, "result should be correct");
     assert_eq!(result[1023], 0x42, "result should be correct");
 }
+
+#[derive(Default)]
+pub struct TestErcToken {}
```

Comment:
This is maybe more for a follow-up, potentially it should go even in a separate repo ([sapphire-paratime](https://github.com/oasisprotocol/sapphire-paratime)?), but we should consider running some ERC-20 token test cases against our precompile implementation?

e.g. https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/test/token/ERC20